dataset/all_types/duckdb_nested_types.csv

```diff
@@ -0,0 +1,3 @@
+int_array,double_array,date_array,timestamp_array,timestamptz_array,varchar_array,nested_int_array,struct,struct_of_arrays,array_of_structs,map,union,fixed_int_array,fixed_varchar_array,fixed_nested_int_array,fixed_nested_varchar_array,fixed_struct_array,struct_of_fixed_array,fixed_array_of_int_list,list_of_fixed_int_array
+[],[],[],[],[],[],[],"{'a': NULL, 'b': NULL}","{'a': NULL, 'b': NULL}",[],{},Frank,"[NULL, 2, 3]","[a, NULL, c]","[[NULL, 2, 3], NULL, [NULL, 2, 3]]","[[a, NULL, c], NULL, [a, NULL, c]]","[{'a': NULL, 'b': NULL}, {'a': 42, 'b': 🦆🦆🦆🦆🦆🦆}, {'a': NULL, 'b': NULL}]","{'a': [NULL, 2, 3], 'b': [a, NULL, c]}","[[], [42, 999, NULL, NULL, -42], []]","[[NULL, 2, 3], [4, 5, 6], [NULL, 2, 3]]"
+"[42, 999, NULL, NULL, -42]","[42.0, nan, inf, -inf, NULL, -42.0]","[1970-01-01, infinity, -infinity, NULL, 2022-05-12]","[1970-01-01 00:00:00, infinity, -infinity, NULL, 2022-05-12 16:23:45]","[1969-12-31 19:00:00-05, infinity, -infinity, NULL, 2022-05-12 19:23:45-04]","[🦆🦆🦆🦆🦆🦆, goose, NULL, ]","[[], [42, 999, NULL, NULL, -42], NULL, [], [42, 999, NULL, NULL, -42]]","{'a': 42, 'b': 🦆🦆🦆🦆🦆🦆}","{'a': [42, 999, NULL, NULL, -42], 'b': [🦆🦆🦆🦆🦆🦆, goose, NULL, ]}","[{'a': NULL, 'b': NULL}, {'a': 42, 'b': 🦆🦆🦆🦆🦆🦆}, NULL]","{key1=🦆🦆🦆🦆🦆🦆, key2=goose}",5,"[4, 5, 6]","[d, e, f]","[[4, 5, 6], [NULL, 2, 3], [4, 5, 6]]","[[d, e, f], [a, NULL, c], [d, e, f]]","[{'a': 42, 'b': 🦆🦆🦆🦆🦆🦆}, {'a': NULL, 'b': NULL}, {'a': 42, 'b': 🦆🦆🦆🦆🦆🦆}]","{'a': [4, 5, 6], 'b': [d, e, f]}","[[42, 999, NULL, NULL, -42], [], [42, 999, NULL, NULL, -42]]","[[4, 5, 6], [NULL, 2, 3], [4, 5, 6]]"
```

Comment:
Always add a newline at the end of files.

Reply:
added